src/core_modules/capture-core/reducers/descriptions/teiSearch.reducerDescription.js

```diff
@@ -35,7 +35,7 @@ export const teiSearchDesc = createReducerDescription({
     [teiSearchActionTypes.SEARCH_TEI_RESULT_RETRIEVED]: (state, action) => {
         const data = action.payload.data;
         const searchId = action.payload.searchId;
-        const teis = data.trackedEntityInstanceContainers ? data.trackedEntityInstanceContainers.map(t => ({ id: t.id, values: t.values })) : [];
```

Comment:
removing this will allow us to also pass the `tei` object key-value pair, which we can then use in order to see if a user has active enrollments etc etc.